src/main/java/com/shrek/task00001/Main.java

```diff
@@ -9,10 +8,66 @@
  */
 public class Main {
     public String getWinner(String[] runners, int[] times) {
-        throw new NotImplementedException();
+
+        String winnerName;
+        if (times.length == 0) {
+            return null;
+        }
+        int min1 = times[0];
+
+        for (int i = 0; i <= times.length - 1; i++) {
+            if (times[i] < min1) {
+                min1 = times[i];
+            }
+        }
+        int winnerPos = 0;
+        for (int pos = 0; pos <= times.length - 1; pos++) {
```

Comment:
No no no. Redundant cycle. You can find the index by only one cycle. Just store it and update with min value